src/bootstrap.rs

```diff
@@ -138,9 +138,8 @@ Private functions
 async fn launch_movie_sample(cx: app::Context) -> Result<(), DyneinBootstrapError> {
     println!(
         "\
-Bootstrapping - dynein will creates 'Movie' table used in public tutorials:
-e.g. https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.NodeJs.02.html
-     https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.Ruby.02.html
+Bootstrapping - dynein will creates 'Movie' table with official 'Movie' sample data:
+https://raw.githubusercontent.com/awsdocs/aws-doc-sdk-examples/main/resources/sample_files/movies.json
```

Comment:
We can change this URL into the permalink. But, this is just a comment, so current URL is maybe preferable due to easy-to-understand. Please select your preferable choice.

Reply:
change to dynein README.md link